modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -565,17 +565,25 @@
 				if( defaultLangKey == 'None' ){
 					return ;
 				}
+				if ( mw.isIOS() ) {
```

Comment:
@GuyLivni not sure we want this for ipad as well, cause in ipad we might show our own SRT rendered captions.